pkgs/development/libraries/gtk/4.x.nix

```diff
@@ -37,6 +33,7 @@
 , x11Support ? stdenv.isLinux
 , waylandSupport ? stdenv.isLinux
 , libGL
+, vulkanSupport ? false
```

Comment:
Why is `vulkanSupport` disabled by default, when it was previously enable when not on Darwin?

Reply:
https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/3391/diffs#ca9bb7eff80503c97c83505e8acea4002fd87ac6_58_54 it is experimental (and can cause crashes in inspector, according to the commit message from jtojnar).

Reply:
Worth a comment I guess.